Organic/AmpAdsInjector.php

```diff
@@ -69,13 +69,44 @@ private function injectConnatix() {
             return;
         }
 
+        $targeting = $this->organic->getTargeting();
+        $section = implode( ',', $targeting['sections'] ?: [] );
+        $keywords = implode( ',', $targeting['keywords'] ?: [] );
+        $gamPageId = $targeting['gamPageId'];
+        $externalId = $targeting['gamExternalId'];
+        $macros = htmlspecialchars(
+            json_encode(
+                [
+                    'cust_params' => http_build_query(
+                        [
+                            'site' => $this->organic->siteDomain,
+                            'targeting_article' => $externalId,
+                            'targeting_section' => $section,
+                            'targeting_keyword' => $keywords,
+                            'article' => $gamPageId,
+                        ]
+                    ),
+                    'article' => $gamPageId,
+                    'category' => $section,
+                    'keywords' => $keywords,
+                ]
+            ),
+            ENT_QUOTES,
+            'UTF-8'
+        );
+
         $psid = $this->organic->getConnatixPlayspaceId();
         $player = "
             <amp-connatix-player
                 data-player-id=\"ps_$psid\"
+                data-param-playback-mode=\"1\"
```

Comment:
Do we really need to explicitly set playback mode?
It's not set in regular Connatix injection code, and I believe in this case playback mode is controlled by configuration in Connatix Portal

Reply:
Ok